src/components/app/menu/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Component, h } from "@stencil/core"
+
+@Component({
+	tag: "smoothly-app-menu",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class SmoothlyAppMenu {
+	render() {
+		return (
+			<ul class="appmenu">
+				<li></li>
+				<li></li>
+				<li></li>
+			</ul>
```

Comment:
why is there a list with three empty list items?

Reply:
THis was a component that i started to create but was not able to finish it.